app/presenters/order_mailchimp_presenter.rb

```diff
@@ -34,19 +43,19 @@ def user
             last_name: order.bill_address&.last_name || "",
             email_address: order.email || "",
             opt_in_status: false,
-            address: address
+            address: address(order.shipping_address)
           }
         end
       end
 
-      def address
-        return {} unless order.shipping_address
+      def address(address)
+        return {} unless address
 
-        AddressMailchimpPresenter.new(order.shipping_address).json
+        AddressMailchimpPresenter.new(address).json
       end
 
-      def processed_at
-        { processed_at_foreign: order.completed_at.strftime("%Y%m%dT%H%M%S") }.as_json
+      def order_address(address)
```

Comment:
could you use more descriptive name for method. I cannot catch the difference between address and order_address